docs/user/ensure_your_experiments_are_reproducible.md

```diff
@@ -0,0 +1,46 @@
+# Ensure your experiments are reproducible
+Ensure the reproducibility of your experiments with comprehensive launcher files. Launcher files are used to initialize the an algorithm, its components (policy, replay buffer, etc.), and tools for managing experiments.
+
+There are some important take-aways from this example launcher file.
+
+```python
+from garage import wrap_experiment
+from garage.envs import GarageEnv
+from garage.experiment import deterministic, LocalRunner
+import a.b.c
+
+@wrap_experiment(wrap_experiment_args, wrap_experiment_kwargs)
+def experiment(ctxt=None, seed):
```

Comment:
i think you should tell them to put their important hyperparameters as arugments to this function, so that `wrap_experiment` will document them.